src/ansys/fluent/core/solver/flversion.py

```diff
@@ -0,0 +1,3 @@
+from ansys.fluent.core.utils.fluent_version import FluentVersion
+
+FluentVersion
```

Comment:
@ansaboutin is this line required?

Reply:
@seanpearsonuk Without the `FluentVersion` line the pre-commit check will not let me have an unused import. Is there a better way to get around it?

Reply:
@ansaboutin 
Try `# noqa: F401` at the end of the import line